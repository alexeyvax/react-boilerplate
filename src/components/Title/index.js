import React from 'react';
import propTypes from 'prop-types';

import theme from './theme.scss';

const Title = ({ name }) => (
  <h1 className={theme.test}>{`Hello ${name}`}</h1>
);

Title.propTypes = {
  name: propTypes.string.isRequired,
};

export default Title;
