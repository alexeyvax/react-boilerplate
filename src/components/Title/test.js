import React from 'react';
import renderer from 'react-test-renderer';

import Title from '.';

it('renders correctly Title component', () => {
  const tree = renderer
    .create(<Title name="Test-name" />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
