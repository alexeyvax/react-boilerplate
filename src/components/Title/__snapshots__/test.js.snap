// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Title component 1`] = `
<h1
  className="test"
>
  Hello Test-name
</h1>
`;
